plugins/confluence-plugin/src/components/PageContent.tsx

```diff
@@ -0,0 +1,71 @@
+// eslint-disable-next-line @typescript-eslint/consistent-type-imports
+import React, { useState, useEffect } from "react";
+// eslint-disable-next-line @typescript-eslint/consistent-type-imports
+// import React from "react";
+import { isEmpty, isNil } from "lodash";
+import "../baseStyles.css";
+import {
+  Box,
+  Text,
+  usePluginContext,
+} from "@cortexapps/plugin-core/components";
+import { getEntityYaml } from "../api/Cortex";
+import { getConfluenceDetailsFromEntity } from "../lib/parseEntity";
+
+const baseJIRAUrl = "https://cortex-se-test.atlassian.net";
```

Comment:
Updating this should be in the plugin's README as instructions for users as to how to point it at their own Jira/Confluence instance. Also, shouldn't this be
```suggestion
const baseConfluenceUrl = "https://cortex-se-test.atlassian.net";
```
?

Lastly, Jira hasn't been stylized in all caps since 2017 😛